compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -104,10 +106,36 @@ void insert_scale_zp(luci::CircleNode *node, float scale, int64_t zp)
   qparam->zerop.push_back(zp);
 }
 
+void run_phase(loco::Graph *g, Type quantized_dtype, Granularity granularity)
```

Comment:
Test codes did not correctly set dtypes of nodes, so I use `Phase` including `CircleTypeInferencePass` for setting them at once.

Previously, only `QuantizeWithMinMaxPass` was invoked.

Reply:
> Test codes did not correctly set dtypes of nodes

Q) plan to fix test codes?
Q) if fix test codes, can we remove this?
If either is not yes, can you add a NOTE about this situation?

Reply:
Ah, this PR applies fix :)